infra/terraform/modules/data_access/teams/teams.py

```diff
@@ -38,5 +38,96 @@ def create_team_bucket(event, context):
     )
 
 
+def create_team_bucket_policies(event, context):
+    """
+    Creates the policies for the team S3 bucket:
+      - a "readonly" policy and
+      - a "readwrite" policy
+
+    event = {"team": {"slug": "justice-league"}}
+    """
+    bucket = bucket_name(event["team"]["slug"])
+
+    create_policy(
+        name="{}-readonly".format(bucket),
+        bucket=bucket,
+        readwrite=False,
+    )
+
+    create_policy(
+        name="{}-readwrite".format(bucket),
+        bucket=bucket,
+        readwrite=True,
+    )
+
+
+def create_policy(name, bucket, readwrite):
+    LOG.debug("Creating '{}' policy".format(name))
+
+    policy_document = get_policy_document(bucket, readwrite)
```

Comment:
I'd be tempted to inline this rather than assign to a variable - it's not a long line or difficult to understand